lib/koans/15_processes.ex

```diff
@@ -17,9 +17,20 @@ defmodule Processes do
     assert is_pid(self()) == ___
   end
 
-  koan "New processes are spawned functions" do
+  koan "Like our current process, new processes are spawned functions" do
     pid = spawn(fn -> nil end)
 
+    assert is_pid(pid) == ___
```

Comment:
I wonder if it would help to really drive that point home if you included two assertions here:

```elixir
assert is_pid(self) == true
assert is_pid(value) == ___
```

You might also rename the return value of the call to spawn/1 to something more generic as to not give it away. What do you think?